From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeff Genovy <29107334+jefgen@users.noreply.github.com>
Date: Fri, 20 Nov 2020 17:30:47 -0800
Subject: MSFT-PATCH: Modify make dist for MS-ICU GitHub paths.

Modify `make dist` paths for the MS-ICU GitHub path location, plus omit
the Doxygen docs as we don't currently build the docs.

diff --git a/icu4c/source/config/dist.mk b/icu4c/source/config/dist.mk
index d82863221b..8399201eee 100644
--- a/icu4c/source/config/dist.mk
+++ b/icu4c/source/config/dist.mk
@@ -37,7 +37,9 @@ DISTY_DATA_ZIP=$(DISTY_FILE_DIR)/$(DISTY_PREFIX)-$(DISTY_VER)-$(GITVER)-data.zip
 DISTY_DAT:=$(firstword $(wildcard data/out/tmp/icudt$(SO_TARGET_VERSION_MAJOR)*.dat))
 
 DISTY_FILES_SRC=$(DISTY_FILE_TGZ) $(DISTY_FILE_ZIP)
-DISTY_FILES=$(DISTY_FILES_SRC) $(DISTY_DOC_ZIP)
+# MSFT-Change: We only want the tgz for now, as we don't currently build the docs.
+#DISTY_FILES=$(DISTY_FILES_SRC) $(DISTY_DOC_ZIP)
+DISTY_FILES=$(DISTY_FILES_SRC)
 # colon-equals because we want to run this once!
 EXCLUDES_FILE:=$(shell mktemp)
 
@@ -65,7 +67,8 @@ $(DISTY_FILE_TGZ) $(DISTY_FILE_ZIP) $(DISTY_DATA_ZIP):  $(DISTY_DAT) $(DISTY_TMP
 	@echo Export icu4c@$(GITVER) to "$(DISTY_TMP)/icu"
 	-$(RMV) $(DISTY_FILE) $(DISTY_TMP)
 	$(MKINSTALLDIRS) $(DISTY_TMP)
-	( cd $(ICU4CTOP)/.. && git archive --format=tar --prefix=icu/ HEAD:icu4c/ ) | ( cd "$(DISTY_TMP)" && tar xf - )
+	# MSFT-Change: Adjust the path for the GitHub repo location.
+	( cd $(ICU4CTOP)/../.. && git archive --format=tar --prefix=icu/ HEAD:icu/icu4c/ ) | ( cd "$(DISTY_TMP)" && tar xf - )
 	( cd $(DISTY_TMP)/icu/source ; zip -rlq $(DISTY_DATA_ZIP) data )
 	$(MKINSTALLDIRS) $(DISTY_IN)
 	echo DISTY_DAT=$(DISTY_DAT)
